os/Kconfig

```diff
@@ -39,6 +39,12 @@ config DEFAULT_SMALL
 		have already been selected in your configuration file.  This applies
 		only to new settings that require a default value.
 
+config DOCKER_VERSION
+	string "Docker Image Version"
+	default "1.5.8"
+	---help---
+		This config indicates docker image version.
```

Comment:
Could you add more explanation for this? like `Different chipset requires different toolchain so that this config provides docker version selection for this.`
You should rewrite my sentence with proper English grammar.

Reply:
I am not sure, but we use 6.3.1 for RTL8730E until now. 2.0.0 docker version needs to enable c++ exception on the loadable build. We need to check where we leave the NOTE Prabu said.

Reply:
Done